src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -110,5 +112,22 @@ public Task BytesMetrics()
 
             return Task.CompletedTask;
         }
+
+        [RavenAction("/databases/*/periodic-export-stats", "GET")]
```

Comment:
Yes, we are changing it, see 
http://issues.hibernatingrhinos.com/issue/RavenDB-6796

The reasonin is that in a cluster, we want to be able to more easily work with restoring from different servers.